### ISSUE TITLE
MegaApiClient.dll depends on old NewtonSoft.Json.dll

### ISSUE THREAD

**Issue by mike101200** [duplicati/duplicati#1890]
### Version info
**Duplicati Version:**  2.0.1.18_canary_2016-08-10
**Operating System:** Windows 10 x64
**Backend:** mega.co.nz

### Bug description
When I try to run a backup to mega.co.nz it end the process at the beginning

### Steps to reproduce
- I updated to latest version  2.0.1.18_canary_2016-08-10 (from 2.0.1.17)
- I try to run a backup task that was working before the update

**Actual result:** The backup is not working
**Expected result:** The backup process should have ended with success

### Screenshots
![duplicati error](https://cloud.githubusercontent.com/assets/1726595/17643572/de62228c-6176-11e6-8b84-dcaf00e4ceac.png)

### debug log
Fatal error
System.IO.FileLoadException: Could not load file or assembly 'Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)

File name: 'Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed'

   at Duplicati.Library.Main.BackendManager.List()

   at Duplicati.Library.Main.Operation.FilelistProcessor.RemoteListAnalysis(BackendManager backend, Options options, LocalDatabase database, IBackendWriter log)

   at Duplicati.Library.Main.Operation.FilelistProcessor.VerifyRemoteList(BackendManager backend, Options options, LocalDatabase database, IBackendWriter log)

   at Duplicati.Library.Main.Operation.BackupHandler.PreBackupVerify(BackendManager backend)

   at Duplicati.Library.Main.Operation.BackupHandler.Run(String[] sources, IFilter filter)

WRN: Assembly binding logging is turned OFF.

To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.

Note: There is some performance penalty associated with assembly bind failure logging.

To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].

**Comment by kenkendk** [comment]
This is identified by @mnaiman to be caused by the MegaApiClient.dll file depending on NewtonSoft.Json.dll version 6, but having Duplicati shipped with a newer version.

**Comment by mnaiman** [comment]
There is a solution that at least for login works (cant test, I dont use mega).
Change Duplicati.Server.exe.config (or whatever exe you are using eg. commandline) to have this inside:
```
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
         <bindingRedirect oldVersion="1.0.0.0-8.0.0.0" newVersion="9.0.0.0"/>
       </dependentAssembly>
     </assemblyBinding>
```
At the end it will look like this:
```
<?xml version="1.0"?>
<configuration>
  <runtime>
    <loadFromRemoteSources enabled="true"/>

     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
         <bindingRedirect oldVersion="1.0.0.0-8.0.0.0" newVersion="9.0.0.0"/>
       </dependentAssembly>
     </assemblyBinding>
   
  </runtime>
  <startup useLegacyV2RuntimeActivationPolicy="true"><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5"/></startup>
</configuration>
```

**Comment by kenkendk** [comment]
I have fixed this with cac13217a664c8846656d90a380d4db4b4a26c94.

The assembly redirects are now installed as part of building the package.
